src/suparnaNJune20/Bank.java

```diff
@@ -0,0 +1,72 @@
+package suparnaNJune20;
+
+public class Bank {
+
+	static double atmBal=100000;
```

Comment:
Do not hard code amount balance, you should write setdata() method for this.

Reply:
done